js/ui/html_editor/modules/tableResizing.js

```diff
@@ -0,0 +1,524 @@
+                positionCoordinateName: 'y'
+            };
+        } else {
+            return {
+                lineResizerClass: DX_COLUMN_RESIZER_CLASS,
+                getSizeFunction: 'outerWidth',
+                positionCoordinate: this.editorInstance.option('rtlEnabled') ? 'right' : 'left',
+                positionStyleProperty: 'width',
+                positionCoordinateName: 'x'
+            };
+        }
+    }
+
+    _updateFrameSeparators(frame, direction) {
+        const $determinantElements = this._getTableDeterminantElements(frame.$table, direction);
+        const determinantElementsCount = $determinantElements.length;
+        const determinantElementsSeparatorsCount = determinantElementsCount - 1;
+        const directionInfo = this._getDirectionInfo(direction);
+        const lineSeparators = frame.$frame.find(`.${directionInfo.lineResizerClass}`);
+        const styleOptions = {
+            transform: 'none'
+        };
+
+        let currentPosition = 0;
+        for(let i = 0; i <= determinantElementsSeparatorsCount; i++) {
+            currentPosition += $determinantElements.eq(i)[directionInfo.getSizeFunction]();
+
+            if(!isDefined(lineSeparators[i])) {
+                lineSeparators[i] = $('<div>')
+                    .addClass(directionInfo.lineResizerClass)
+                    .appendTo(frame.$frame)
+                    .get(0);
+            }
+
+            const $currentLineSeparator = $(lineSeparators[i]);
+
+            this._removeDraggable($currentLineSeparator, directionInfo.lineResizerClass);
+
+            styleOptions[directionInfo.positionCoordinate] = currentPosition - DRAGGABLE_ELEMENT_OFFSET;
+            $($currentLineSeparator).css(styleOptions);
+
+            const attachSeparatorData = {
+                lineSeparator: lineSeparators[i],
+                index: i,
+                $determinantElements,
+                frame,
+                direction
+            };
+            this._attachColumnSeparatorEvents(attachSeparatorData);
+        }
+    }
+
+    _getTableDeterminantElements($table, direction) {
+        if(direction === 'vertical') {
+            return $table.find('td:first-child');
+        } else {
+            const $theadElements = $table.find('th');
+            if($theadElements.length) {
+                return $theadElements;
+            } else {
+                return $table.find('tr').eq(0).find('td');
+            }
+        }
+    }
+
+    _attachColumnSeparatorEvents(options) {
+        eventsEngine.on(options.lineSeparator, POINTERDOWN_EVENT, () => {
+            this._createDraggableElement(options);
+        });
+    }
+
+    _dragStartHandler({ $determinantElements, index, frame, direction }) {
+        const directionInfo = this._getDirectionInfo(direction);
+
+        this._fixColumnsWidth(frame.$table);
+        this._startLineSize = parseInt($($determinantElements[index])[directionInfo.getSizeFunction]());
```

Comment:
mb we should create a function for size getting to avoid copypaste and difficult contructions?